packages/runtime/src/helpers/verification.ts

```diff
@@ -210,7 +213,7 @@ export const warnForProblematicUserRewrites = ({
 
 export const warnForRootRedirects = ({ appDir }: { appDir: string }) => {
   if (existsSync(join(appDir, '_redirects'))) {
-    console.log(
+    logger.info(
       yellowBright(
```

Comment:
do we need the colours still, or does winston provide different colours for different log levels?

Reply:
Good idea. It does allow you to colour them but we'd have to add an additional config.

https://github.com/winstonjs/winston#colorizing-standard-logging-levels

Should we do this and remove our use of `chalk` or do we want to leave it as is so it matches the current behaviour?

Reply:
Maybe in a small follow up PR once the new logger is up and running. If it's not a big lift, go for it now.

Reply:
I took a look and we'd have to ensure we used the correct log levels. In some cases, we're changing colours but still using `logger.info`. Might be worth shipping this and then updating the log levels if we're considering some of them as different messages.